molo/core/migrations/0078_add_translation_fields.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-09-03 12:40
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0003_auto_20180831_1312'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='articlepage',
+            options={'ordering': ('-latest_revision_created_at',), 'verbose_name': 'Article'},
+        ),
+        migrations.AddField(
+            model_name='articlepage',
+            name='language',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='core.SiteLanguage'),
+        ),
+        migrations.AddField(
+            model_name='articlepage',
+            name='translated_pages',
+            field=models.ManyToManyField(blank=True, null=True, related_name='_articlepage_translated_pages_+', to='core.ArticlePage'),
+        ),
+        migrations.AddField(
+            model_name='sectionpage',
+            name='language',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='core.SiteLanguage'),
+        ),
+        migrations.AddField(
+            model_name='sectionpage',
+            name='translated_pages',
+            field=models.ManyToManyField(blank=True, null=True, related_name='_sectionpage_translated_pages_+', to='core.SectionPage'),
+        ),
+        migrations.AddField(
+            model_name='tag',
+            name='language',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='core.SiteLanguage'),
+        ),
+        migrations.AddField(
+            model_name='tag',
+            name='translated_pages',
+            field=models.ManyToManyField(blank=True, null=True, related_name='_tag_translated_pages_+', to='core.Tag'),
+        ),
+        migrations.AlterField(
+            model_name='sitesettings',
+            name='default_service_directory_radius',
+            field=models.PositiveSmallIntegerField(blank=True, help_text='When set this will enable service directory radius filter as the set value for the default radius value', null=True, verbose_name='Default Service Directory Radius'),
+        ),
```

Comment:
What is the above migration for? Please find out where that change comes from. If this is a migration that someone else forgot to add to their work please make a separate migration and PR for it.